build.gradle

```diff
@@ -1,11 +1,13 @@
 buildscript {
+	ext.kotlin_version = '1.2.31'
```

Comment:
I would prefer `kotlinVersion` for consistency with all other similar variables.

Reply:
Good catch - this was generated by IDEA 😄

Reply:
Ha! Don't tell me that STS doesn't support Kotlin yet! :wink:

Reply:
It's buggy - I figured the support in IDEA would be first class.